core/WFCORE-6960_Simple_config_export_for_a_server_as_an_attachment_for_standalone_or_domain_mode.adoc

```diff
@@ -2,8 +2,9 @@
 categories:
  - core
  - management
+stability-level: default
 ---
-= [Community] Simple config export for a server as an attachment for standalone or domain mode 
```

Comment:
Note that the '[Community]' here was a mistake, probably cut and paste from the earlier proposal for the community stability feature.